README.md

```diff
@@ -0,0 +1,45 @@
+# Guide
+
+A website to teach Wagtail CMS to content editors, moderators and administrators.
+
+Wagtail guide brings you:
+- Tutorials, 
+- How-to, 
+- Reference 
+- Background information
+
+## Google Summer of Code
+
+This project is a part of Google Summer of Code 2022 under the organization [**Wagtail**](https://wagtail.org/).
+
+## Development
+
+### Frontend
+
+- Setup the appropriate version of node.    
+`nvm use`   
+The node version shoule be `v16.*`. You can check it by running 
```

Comment:
shoule isn't a word ;). I also like instructions to be as compact as possible. People do not read. Maybe: "Check the Node version number. It should be `v16.*`"